pkg/events/component.go

```diff
@@ -40,6 +42,23 @@ func (ec *component) reportEvent(ctx context.Context, apiCall string, values ...
 
 }
 
+// Get actions
+func (ec *component) GetActions(ctx context.Context) ([]api.ActionRepresentation, error) {
```

Comment:
Could be part of the common lib

Reply:
I agree this method is very similar for each Component but a simple and elegant way to do it ...
Would you have any idea ?

Reply:
Let's say we will have a look at this when we'll have time for this... (it almost means never)